shared/chat/conversation/input-area/normal/platform-input.native.js

```diff
@@ -174,6 +181,10 @@ class PlatformInput extends Component<PlatformInputProps & OverlayParentProps, S
             visible={this.props.showingMenu}
           />
         )}
+        <IsTyping
+          style={collapseStyles([styles.typing, {opacity: this.props.typing.size > 0 ? 1 : 0}])}
```

Comment:
likely want to just do
```
collapseStyles([styles.typing, this.props.typing.size && styles.visibleTyping}])
```

and make styles.typing have opacity: 0 and visibleTyping have opacity:1